### DISCUSSION TITLE
加载脚本后，浏览器打开pdf文件出现乱码，关闭则正常；请问如何解决？

### DISCUSSION THREAD

**Discussion by kinkir** [F9y4ng/GreasyFork-Scripts#392] [问答专区(Question / Answer)]
如题所述，chrome系加载了字体渲染（自用脚本）,出现乱码，以夸克为例
![Screenshot 2024-10-28 003655](https://github.com/user-attachments/assets/561acba6-9d99-4b20-893f-336e845029cb)
在没有使用脚本字体渲染（自用脚本）的firefox上也都正常
![Screenshot 2024-10-28 004410](https://github.com/user-attachments/assets/78e52546-326d-4545-9dba-dbeba5943934)

**Comment by F9y4ng** [comment]
Duplicate of #373
